py/strato/common/log/configurelogging.py

```diff
@@ -72,6 +72,11 @@ def addFileHandler(name, path):
     logging.info("Logging started for %s" % name)
 
 
+def changeHandlerLogLevelbyHandlerType(logger, logLevel, handlerType=None):
+    """ if not specifying handlers type , change log logleve to all """
```

Comment:
Ohh. Missed it. So please make the variable more descriptive. None means nothing, not everything. It's not clear enough